tick/base/simulation/simu.py

```diff
@@ -45,6 +44,9 @@ class Simu(ABC, Base):
         },
         "_time_start": {
             "writable": False
+        },
+        "dtype": {
+            "writable": True
```

Comment:
`dtype` should be either in parameter (if it's a constructor argument) or attributes and hence does not need to appear in `_attrinfos `

Reply:
I severely dislike comments that have any functionality - it's defeats the purpose of comments. And this is what I found that worked

Reply:
seems to work as an attribute - still think it's gross tho